backend/python/app/__init__.py

```diff
@@ -36,7 +37,14 @@ def create_app(config_name):
     mysql = MySQL()
     app.config.from_object(app_config[config_name])
 
-    app.config["CORS_ORIGINS"] = ["http://localhost:3000"]
+    if os.getenv("IS_PREVIEW_DEPLOY", "False") == "True":
```

Comment:
PR firebase URLs are different per PR--there are two running heroku deployments; one that accepts any origin that starts with `https://planet-read-uwbp` and one that only accepts the staging URLs